chapters/substitution.adoc

```diff
@@ -2,21 +2,36 @@
 .Definition
 Robots are substituted by the <<Robot Handler, robot handler>> of the respective team. No other team member is allowed to take robots out or put robots in.
 
-. A <<Robot Handler, robot handler>> who's team is marked for robot substitution performs at least one of the following operations in the given order.
-.. Put robots in. The maximum allowed number of robots of the team on the field must not be exceeded.
-.. Take robots out. If a robot that is taken out is closer than 1 meter to the intersection of the halfway line with one of the touch lines, another robot can be put in right away. Otherwise the robot handler has to wait until the next substitution opportunity.
+. The <<Robot Handler, robot handler>> should prefer to use long sleeves and colors that won't interfere with the vision system.
+
+. Robots can always be substituted during game play if the following specifications are respected:
+
+.. Is at least partially inside the <<Field Surface, field margin>>
+.. At a distance from the halfway line that must not exceed 30 centimeters
+.. The ball must be at least 2 meters away from the halfway line
+
+. If the previous specifications are respected, the <<Robot Handler, robot handler>> is allowed to perform the following operations.
+... Put robots in, without notifying the <<Referee, referee>>.
+... Take robots out, without notifying the <<Referee, referee>>.
+
+. Given that robots can be put in at any time, while respecting the previous specifications, on a game stoppage robots can only be removed.
```

Comment:
That sounds a bit complicated. What about:

> Additionally, robots can be taken out from any position during game stoppage

Reply:
Yeah, I agree, thanks :sweat_smile: